recipes/symfit/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "symfit" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 431e5c0ae835eda1f40c5935ac13c108c03d9dbf8aed342c20fe627e2094996e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  
+requirements:
+  host:
+    - pip
+    - python
```

```suggestion
    - python
    - pbr >=1.9
```

judging from https://github.com/tBuLi/symfit/blob/0.4.6/setup.py you also need this